### ISSUE TITLE
Wipe after a period of inactivity

### ISSUE THREAD

**Issue by Gaff** [Zyg0te/Locker#5]
Another feature I'd be interested in - could you make the phone wipe if the user has not unlocked for > 48 hours or similar?

**Comment by Zyg0te** [comment]
Interesting feature suggestion. A sort of proof-of-life timer then? If no unlock has happened within the specified timeframe, the phone should wipe?

**Comment by Gaff** [comment]
Exactly. The idea being that if the phone is lost / stolen (defined as not-being-unlocked-for-a-time-period) it should wipe.

**Comment by HostFat** [comment]
I agree that this very good and needed feature :)
